nexus-and-alo/milestones.md

```diff
@@ -0,0 +1,50 @@
+# Milestones
+
+## MVP - Phase 0 (Namespace to Namespace)
+
+**Rationale:** Keeps requirements as minimal as possible and makes the primary focus unblocking X-Namespace calls.
+
+**Requirements:**
+
+* Existing Temporal users
+* Only target scenarios where Temporal Namespace is calling to another Temporal Namespace
```

Comment:
```suggestion
* Only target scenarios where Temporal Namespace is calling to another Temporal Namespace. To be explicit, we will not initially support client-to-Temporal-ALO calls. 
```

Reply:
I was hoping we would support this in the initial version

Reply:
I will say its not a requirement but if its possible within scope of mvp I'm not opposed.